packages/core/src/base/client.ts

```diff
@@ -42,9 +43,7 @@ export class ClientConfig {
 
 const DEFAULT_CLIENT_CONFIG: ClientConfig = {
   // Enables cacache for local caching in temp dir by default, unsafeCleanup == rm -rf on exit
-  cacheManager: new DefaultCache(
-    tmp.dirSync({ prefix: "cache-", unsafeCleanup: true }).name
-  ),
+  cacheManager: new CacheManagerKeyv( { keyvStore: new Map() }),
```

Comment:
Does keyv still do expiry the correct way using this?

Reply:
Setting expiry is still a function of the cache-manager. This uses the cache manager @bredmond-sf wrote for Redis.

Reply:
Discussed and let's use quick-lru for the store.

Reply:
Done